store/sql_post_store.go

```diff
@@ -299,6 +299,69 @@ func (s SqlPostStore) GetPosts(channelId string, offset int, limit int) StoreCha
 	return storeChannel
 }
 
+func (s SqlPostStore) GetPostsSince(channelId string, time int64) StoreChannel {
+	storeChannel := make(StoreChannel)
+
+	go func() {
+		result := StoreResult{}
+
+		var posts []*model.Post
+		_, err := s.GetReplica().Select(&posts,
+			`(SELECT
+			    *
+			FROM
+			    Posts
+			WHERE
+			    (UpdateAt > :Time
+			        AND ChannelId = :ChannelId)
+			LIMIT 100)
+			UNION
+			(SELECT
+			    *
+			FROM
+			    Posts
+			WHERE
+			    Id
+			IN
+			    (SELECT * FROM (SELECT
+			        RootId
+			    FROM
+			        Posts
+			    WHERE
+			        UpdateAt > :Time
+			            AND ChannelId = :ChannelId
+			    LIMIT 100) temp_tab))
+			ORDER BY CreateAt DESC`,
+			map[string]interface{}{"ChannelId": channelId, "Time": time})
+
+		if err != nil {
+			result.Err = model.NewAppError("SqlPostStore.GetPostsSince", "We couldn't get the posts for the channel", "channelId="+channelId+err.Error())
+		} else {
+
+			list := &model.PostList{Order: make([]string, 0, len(posts))}
+
+			for _, p := range posts {
+				if p.ImgCount > 0 {
+					p.Filenames = []string{}
+					for i := 0; int64(i) < p.ImgCount; i++ {
+						fileUrl := "/api/v1/files/get_image/" + p.ChannelId + "/" + p.Id + "/" + strconv.Itoa(i+1) + ".png"
```

Comment:
This is to support legacy systems that used the old images, do we want to get rid of it? I can get rid of it from the other GetPosts DB functions too. Although I think that would be better in a different ticket (I can still remove it here if we want)